README.md

```diff
@@ -245,16 +245,18 @@ You should then be able to `govuk-docker build` your service and have confidence
 
 The following sections resolve common installation issues.
 
-#### How to: update Ruby
+#### How to: install/update Ruby
 
-Update Ruby using [brew](https://brew.sh/), then add it to the PATH in your bash config.
+Follow the instructions to install [rbenv](https://github.com/rbenv/rbenv#installation) using [brew](https://brew.sh/). 
```

Comment:
We can replace this sentence with `brew install rbenv` in the code block below, I think.

I'm tempted to also suggest adding it to the [Brewfile](https://github.com/alphagov/govuk-docker/blob/master/Brewfile), but that only runs on `govuk-docker setup` and if you're in this section then I assume that `govuk-docker` isn't running at all.

Reply:
There's a few additional steps after `brew install rbenv` to complete installation, including setting up `rbenv` in the shell with an added line for `.bash_profile`, and then running the rbenv-doctor script to validate install.

Can add these all to the code block if appropriate?

Reply:
Now you mention all those steps, I'm going to say let's stick with your original suggestion of following the `rbenv` README. We don't want to duplicate every instruction.

Reply:
Thanks - have fixed-up branch, and will merge in PR.